config/routes.rb

```diff
@@ -11,6 +11,8 @@
   get "/choose", to: "static_pages#choose"
 
   resource :pending_loan, except: [:index, :edit]
+  post :delete_pending_loan, to: 'pending_loans#delete_one'
+  post :update_loan_amount, to: 'pending_loans#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.